core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1360,6 +1360,16 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
     listenerBus.addListener(listener)
   }
 
+  private[spark] override def getExecutorIds(): Seq[String] = {
```

Comment:
A newbie question: if a method has no side effect and return values, do code standard in spark suggest to remove parenthesis in method declaration?